plugins/org.eclipse.sirius.table.ui/src/org/eclipse/sirius/table/ui/tools/internal/editor/provider/DTableColumnLabelProvider.java

```diff
@@ -93,21 +115,17 @@ public DTableColumnLabelProvider(final DColumn column) {
     @Override
     public Color getBackground(final Object element) {
         Color result = null;
-        if (column != null) {
-            Option<DCell> optionalCell = getDCell(element);
-            Option<DTableElementStyle> styleToApply = null;
-            if (optionalCell.some()) {
-                styleToApply = new DCellQuery(optionalCell.get()).getBackgroundStyleToApply();
-            } else if (element instanceof DLine) {
-                styleToApply = TableHelper.getBackgroundStyleToApply((DLine) element, column);
-            }
-            if (styleToApply != null && styleToApply.some()) {
-                final RGBValues rgb = styleToApply.get().getBackgroundColor();
-                if (rgb != null) {
-                    result = VisualBindingManager.getDefault().getColorFromRGBValues(rgb);
-                }
+        
```

Comment:
These lines must be moved in previous commit `Use default selection color for table.`.